db/change_cache.go

```diff
@@ -960,3 +984,24 @@ func (l *SkippedSequenceList) getOlderThan(skippedExpiry time.Duration) []uint64
 	l.lock.RUnlock()
 	return oldSequences
 }
+
+func (c *changeCache) _getOldestPending() uint64 {
```

Comment:
Don't think this method is used any more given the updated handling below - we should remove it.